pype/plugins/standalonepublisher/publish/submit_transcode_deadline.py

```diff
@@ -0,0 +1,298 @@
+import os
+from urllib.parse import urlparse
+import json
+import getpass
+import shutil
+import tempfile
+
+import opentimelineio as otio
+import pyblish.api
+import avalon
+from avalon.vendor import requests
+
+import pype
+import pyblish
+import pypeapp
+import acre
+
+import pype.api
+import pype.lib
+from pype.scripts import transcode
+
+
+class SubmitTranscodeDeadline(pyblish.api.InstancePlugin):
+    """Submit transcode to Deadline."""
+
+    order = pyblish.api.IntegratorOrder - 0.1
+    label = "Submit to Deadline"
+    hosts = ["standalonepublisher"]
+    families = ["transcode"]
+
+    def get_published_path(self, representation, instance):
+        anatomy = instance.context.data["anatomy"]
+        template_data = instance.data.get("anatomyData")
+        template_data["representation"] = representation["name"]
+        template_data["ext"] = representation["ext"]
+        template_data["comment"] = None
+        anatomy_filled = anatomy.format(template_data)
+        template_filled = anatomy_filled["publish"]["path"]
+        published_path = os.path.normpath(template_filled)
+
+        # Ensure extension is correct. Anatomy is confusing name with ext.
+        path, ext = os.path.splitext(published_path)
+        published_path = "{}{}".format(path, representation["ext"])
+
+        return published_path
+
+    def process(self, instance):
+        # Requried environment.
+        PYTHONPATH = ""
+        for module in [pype, pyblish, avalon, pypeapp, acre]:
```

Comment:
Think it was to be explicit in the modules, to prevent implicit modules messing up the job on the farm. Good point about the `clique` module. Will add it.